sys-fs/cryfs/cryfs-9999.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 	# remove tests that require internet access to comply with Gentoo policy
 	sed -i -e '/CurlHttpClientTest.cpp/d' -e '/FakeHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
 
+	# remove non-applicable warning caused by cmake-utils setting NDEBUG
```

Comment:
Err, but the warning is conditional to NDEBUG **not** being set? I'll remove that explanation because the removal itself makes sense.